### DISCUSSION TITLE
move the foldout icon to the left of the menu, not the right for speedier navigation

### DISCUSSION THREAD

**Discussion by sxsws** [directus/directus#14182] [Feature Requests]
Currently, the foldout icon is at the end of each word which means the location for each foldout is in a different position and you have to look for it rather than just line the mouse up and quickly navigate where you want to go.

I think placing it on the left will make a big difference to the navigation.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
